### PR TITLE
Remove parameters from InheritanceDiagram macro calls

### PR DESCRIPTION
**Pull request by Elchi3**
For https://github.com/mdn/yari/pull/5056

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
I looked for `{{InheritanceDiagram(` and found 171 results, 4 more than here:

```
./files/en-us/web/api/websocket/close_event/index.md:{{InheritanceDiagram("CloseEvent")}}
./files/en-us/web/api/websocket/message_event/index.md:{{InheritanceDiagram("MessageEvent")}}
./files/en-us/web/api/usb/connect_event/index.md:{{InheritanceDiagram(500, 70, 50, "USBConnectionEvent")}}
./files/en-us/web/api/usb/disconnect_event/index.md:{{InheritanceDiagram(500, 70, 50, "USBConnectionEvent")}}
```

It looks like they were all added after this PR, in https://github.com/mdn/content/pull/11051  and https://github.com/mdn/content/pull/11234.

So I guess these:

```
./files/en-us/web/api/websocket/close_event/index.md:{{InheritanceDiagram("CloseEvent")}}
./files/en-us/web/api/websocket/message_event/index.md:{{InheritanceDiagram("MessageEvent")}}
```

...are good, because these are new-style event pages that want to pass an explicit argument. But these:

```
./files/en-us/web/api/usb/connect_event/index.md:{{InheritanceDiagram(500, 70, 50, "USBConnectionEvent")}}
./files/en-us/web/api/usb/disconnect_event/index.md:{{InheritanceDiagram(500, 70, 50, "USBConnectionEvent")}}
```

...are not?

File: files/en-us/web/api/htmlmediaelement/onencrypted/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The **`onencrypted`** property of the
 
 This interface inherits from the {{domxref("ExtendableEvent")}} interface.
 
-{{InheritanceDiagram(700, 60, 20)}}
+{{InheritanceDiagram}}
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/htmlmediaelement/onencrypted/index.md]: > I guess that _apart from new-style event pages_, we should not use `InheritanceDiagram` on subpages of interface pages. And when we do use them, for new-style event pages, we need to pass the event argument - it's basically never going to be correct to have no arguments unless the page is directly under /API. Is that right

Yes, exactly right. I removed the {{InheritanceDiagram}} calls from these pages. (These are on-* pages and will be rewritten anyways, though.)

**Comment by Elchi3** [comment]
The Yari PR is ready and this seems ready now, too. I'm marking both as ready for review. It would be nice to merge at the same time.

**Review by wbamberg — APPROVED** [review]
This looks great to me now, thanks for the updates Florian!

**Pull request merged**